tests/sparse_nm_test.py

```diff
@@ -203,6 +195,24 @@ def test_validate_nm_spmm(self):
           dimension_numbers=(((2,), (1,)), ((0,), (0,))),
       )
 
+@jtu.with_config(jax_enable_x64=True)
```

Comment:
Let's not do this, because it's not safe to use X64 locally in general. Instead let's skip tests if x64 is not enabled.

Reply:
Done.